qcfractal/interface/collections/dataset.py

```diff
@@ -263,56 +338,86 @@ def list_records(self,
         ret.sort_index(inplace=True)
         return ret
 
-    def _get_values(self, force: bool = False, **search: Dict[str, Optional[str]]) -> 'DataFrame':
-        """Queries for all history that matches the search
+    def get_values(self,
+                   method: Optional[str] = None,
+                   basis: Optional[str] = None,
+                   keywords: Optional[str] = None,
+                   program: Optional[str] = None,
+                   driver: Optional[str] = None,
+                   name: Optional[str] = None,
+                   native: Optional[bool] = None,
+                   force: bool = False) -> pd.DataFrame:
+        """
+        Obtains values from the known history from the search paramaters provided for the expected `return_result` values.
+        Defaults to the standard programs and keywords if not provided.
+
+        Note that unlike `get_records`, `get_values` will automatically expand searches and return multiple method
+        and basis combinations simultaneously.
 
         Parameters
         ----------
+        method : Optional[str], optional
+            The computational method (B3LYP)
+        basis : Optional[str], optional
+            The computational basis (6-31G)
+        keywords : Optional[str], optional
+            The keyword alias
+        program : Optional[str], optional
+            The underlying QC program
+        driver : Optional[str], optional
+            The type of calculation (e.g. energy, gradient, hessian, dipole...)
+        name : Optional[str], optional
+            Canonical name of the record. Overrides the above selectors.
+        native: Optional[bool], optional
+            True: only include data computed with QCFractal
+            False: only include data contributed from outside sources
+            None: include both
         force : bool, optional
-            Force a query even if the data is already present
-        **search : Dict[str, Optional[str]]
-            History query paramters
+            Data is typically cached, forces a new query if True
 
         Returns
         -------
         DataFrame
-            A DataFrame of the queried values
-
-        Raises
-        ------
-        KeyError
-            If no records match the query
+            A DataFrame of values with columns corresponding to methods and rows corresponding to molecule entries.
+            Contributed (native=False) columns are marked with "(contributed)" and may include units in square brackets
+            if their units differ in dimensionality from the Dataset's default units.
         """
-
-        queries = self.list_records(**search, dftd3=True, pretty=False).reset_index()
-        if queries.shape[0] > 10:
-            raise TypeError("More than 10 queries formed, please narrow the search.")
-
+        spec = locals()
```

Comment:
Kind of resisted something as opaque as locals for some time.

Reply:
LGTM also is not a fan of locals. I'll go ahead and spell it out explicitly.